scripts/globals/summon.lua

```diff
@@ -244,85 +190,85 @@ function AvatarFinalAdjustments(dmg,mob,skill,target,skilltype,skillparam,shadow
 
 
     if (skilltype == dsp.attackType.PHYSICAL and target:hasStatusEffect(dsp.effect.PHYSICAL_SHIELD)) then
-        return 0;
+        return 0
     end
 
     if (skilltype == dsp.attackType.RANGED and target:hasStatusEffect(dsp.effect.ARROW_SHIELD)) then
-        return 0;
+        return 0
     end
 
     -- handle elemental resistence
     if (skilltype == dsp.attackType.MAGICAL and target:hasStatusEffect(dsp.effect.MAGIC_SHIELD)) then
-        return 0;
+        return 0
     end
 
     -- handling phalanx
-    dmg = dmg - target:getMod(dsp.mod.PHALANX);
+    dmg = dmg - target:getMod(dsp.mod.PHALANX)
     if (dmg<0) then
-        return 0;
+        return 0
     end
 
     --handle invincible
     if (target:hasStatusEffect(dsp.effect.INVINCIBLE) and skilltype==dsp.attackType.PHYSICAL) then
-        return 0;
+        return 0
     end
     -- handle pd
     if ((target:hasStatusEffect(dsp.effect.PERFECT_DODGE) or target:hasStatusEffect(dsp.effect.ALL_MISS) )
             and skilltype==dsp.attackType.PHYSICAL) then
-        return 0;
+        return 0
     end
-    
+
     -- Calculate Blood Pact Damage before stoneskin
     dmg = dmg + dmg * mob:getMod(dsp.mod.BP_DAMAGE) / 100
 
     -- handling stoneskin
-    skin = target:getMod(dsp.mod.STONESKIN);
+    skin = target:getMod(dsp.mod.STONESKIN)
     if (skin>0) then
         if (skin >= dmg) then --absorb all damage
-            target:delMod(dsp.mod.STONESKIN,dmg);
+            target:delMod(dsp.mod.STONESKIN,dmg)
             if (target:getMod(dsp.mod.STONESKIN)==0) then
-                target:delStatusEffect(dsp.effect.STONESKIN);
+                target:delStatusEffect(dsp.effect.STONESKIN)
             end
-            return 0;
+            return 0
         else -- absorbs some damage then wear
-            target:delMod(dsp.mod.STONESKIN,skin);
-            target:delStatusEffect(dsp.effect.STONESKIN);
-            return dmg - skin;
+            target:delMod(dsp.mod.STONESKIN,skin)
+            target:delStatusEffect(dsp.effect.STONESKIN)
+            return dmg - skin
         end
     end
 
-    return dmg;
-end;
+    return dmg
+end
 
 -- returns true if mob attack hit
 -- used to stop tp move status effects
 function AvatarPhysicalHit(skill, dmg)
     -- if message is not the default. Then there was a miss, shadow taken etc
-    return skill:getMsg() == dsp.msg.basic.DAMAGE;
-end;
+    return skill:getMsg() == dsp.msg.basic.DAMAGE
+end
 
 function avatarFTP(tp,ftp1,ftp2,ftp3)
     if (tp < 1000) then
-        tp = 1000;
+        tp = 1000
     end
     if (tp >= 1000 and tp < 2000) then
-        return ftp1 + ( ((ftp2-ftp1)/100) * (tp-1000));
+        return ftp1 + ( ((ftp2-ftp1)/100) * (tp-1000))
```

Comment:
Holy parentheses, batman. Let order of operations do it's job.

`ftp1 + (ftp2 - ftp1) / 100 * (tp - 1000)`